base/exports.jl

```diff
@@ -407,6 +407,7 @@ export
     prevpow2,
     prevprod,
     primes,
+    primesmask,
```

Comment:
I was just following the comments in this discussion https://github.com/JuliaLang/julia/issues/11594#issuecomment-109753437

(Also, I'm aware this would need documentation, I just haven't added it yet)